lib/ruby_smb/smb1/packet/nt_create_andx_response.rb

```diff
@@ -0,0 +1,74 @@
+module RubySMB
+  module SMB1
+    module Packet
+
+      # A SMB1 SMB_COM_NT_CREATE_ANDX Response Packet as defined in
+      # [2.2.4.64.2 Response](https://msdn.microsoft.com/en-us/library/ee441612.aspx) and
+      # [2.2.4.9.2 Server Response Extensions](https://msdn.microsoft.com/en-us/library/cc246334.aspx)
```

Comment:
According to the documentation ( [2.2.4.9.2 Server Response Extensions] (https://msdn.microsoft.com/en-us/library/cc246334.aspx) ), word_count should be 0x2A, but there is this comment:

https://msdn.microsoft.com/en-us/library/cc246806.aspx#Appendix_A_49
> Windows-based SMB servers send 50 (0x32) words in the extended response although they set the WordCount field to 0x2A

I am confused, is it something we should care about?

Reply:
I doubt it, since this is a response we can probably just ignore this as long as we are parsing the response correctly

Reply:
Sounds good! (but it is still a scary protocol design :) )